docs/src/pages/premium-themes/paperbase/Navigator.js

```diff
@@ -133,7 +129,6 @@ function Navigator(props) {
                 <ListItemText
                   classes={{
                     primary: classes.itemPrimary,
-                    textDense: classes.textDense,
```

Comment:
What happens when you change this to `dense: classes.textDense` ?

Reply:
Nothing as textDense is {}

Reply:
That’s fine textDense is used in itemPrimary to override the fontSize. It should still work

Reply:
I have updated the source.